### DISCUSSION TITLE
Is it possible to start new players at a different mu?

### DISCUSSION THREAD

**Discussion by joshua-augustinus** [OpenDebates/openskill.py#129] [Q&A]
So one of the main assumptions of this library is that new players start at the average mu. Is it possible to start new players at a lower mu? For example if I join chess.com I could choose a starting ELO that is at a beginner level.

**Comment by vivekjoshy** [comment]
Shifting the `mu` value ($\mu$) has no effect on the rating system since it's just an arbitrary number. Shifting it will only shift the mean. The system starts with an assumption of $\sigma = \frac{\mu}{3}$. To take the default values as an example, at $\mu = 25$ and $\sigma = \frac{25}{3}$ implies the system has ascertained the actual skill of the player could be anywhere within $3$ standard deviations of the mean. In other words, the system has no idea about the skill of the player. Don't compare it to Elo, since it does not have a variance parameter.

**Reply by philihp** [reply]
^yep, this.

If you take a look at [MatchPoker](https://matchpoker.com/the-app/ifmp-ratings) which uses Openskill, they start players at a &mu; of 1200 and a &sigma; of 400. It doesn't really matter what you start at, but you don't want to change it once you do. We picked our defaults so they would be on a familiar scale to TrueSkill ratings.

**Comment by jauggy** [comment]
So one of the issues we have is that when we want to create a balanced team of players, the new players are severely overrated causing the teams to be unbalanced. Is this a problem that has come up?
